Libraries/Components/Touchable/Touchable.js

```diff
@@ -652,12 +698,12 @@ const TouchableMixin = {
     );
   },
 
-  _handleDelay: function(e) {
+  _handleDelay: function(e: LayoutEvent) {
     this.touchableDelayTimeout = null;
     this._receiveSignal(Signals.DELAY, e);
   },
 
-  _handleLongDelay: function(e) {
+  _handleLongDelay: function(e: LayoutEvent) {
```

Comment:
Same deal here. Why is this a `LayoutEvent` and not a `PressEvent`? This function calls `_receiveSignal`, which I think should accept a `PressEvent`.